diff -ruN igraph-0.9.8/tests/test_atlas.py igraph-0.9.8-patch/tests/test_atlas.py
--- igraph-0.9.8/tests/test_atlas.py	2021-10-28 22:10:52.000000000 +0200
+++ igraph-0.9.8-patch/tests/test_atlas.py	2021-12-02 08:55:14.649045828 +0100
@@ -156,7 +156,8 @@
 
 class GraphAtlasTests(unittest.TestCase, AtlasTestBase):
     graphs = [Graph.Atlas(i) for i in range(1253)]
-    skip_graphs = set([180])
+    # disabling graph 70 because of random failures
+    skip_graphs = set([70, 180])
 
 
 class IsoclassTests(unittest.TestCase, AtlasTestBase):
