diff -ruN igraph-0.10.4/setup.py igraph-0.10.4-patch/setup.py
--- igraph-0.10.4/setup.py	2023-01-27 17:38:25.000000000 +0100
+++ igraph-0.10.4-patch/setup.py	2023-06-30 20:19:09.091432620 +0200
@@ -253,7 +253,7 @@
 
         # No need to build tests
         args.append("-DBUILD_TESTING=OFF")
-        
+
         # Set install directory during config step instead of install step in order
         # to avoid having the architecture name in the LIBPATH (e.g. lib/x86_64-linux-gnu)
         args.append("-DCMAKE_INSTALL_PREFIX=" + str(install_folder))
@@ -265,6 +265,9 @@
         # Finally, add the source folder path
         args.append(str(build_to_source_folder))
 
+        print('Running CMake:')
+        print(' '.join(args))
+
         retcode = subprocess.call(args)
         if retcode:
             return False
@@ -288,7 +291,21 @@
         if retcode:
             return False
 
-        for candidate in install_folder.rglob("igraph.pc"):
+        print(install_folder)
+
+        print('glob non-recursive:')
+        for candidate in install_folder.glob("*"):
+            print(candidate)
+
+        print('glob recursive:')
+        for candidate in install_folder.glob("**/*"):
+            print(candidate)
+
+        print('rglob:')
+        for candidate in install_folder.rglob("*"):
+            print(candidate)
+
+        for candidate in install_folder.glob("**/igraph.pc"):
             return self._parse_pkgconfig_file(candidate)
 
         raise RuntimeError(
