diff --git a/msvc/include/config.h b/msvc/include/config.h
index b4f07f49..e967bddb 100644
--- a/msvc/include/config.h
+++ b/msvc/include/config.h
@@ -5,7 +5,9 @@
 /* #undef HAVE_DLFCN_H */
 
 /* Define to 1 if you have the `expm1' function. */
-/* #undef HAVE_EXPM1 */
+#if defined(_MSC_VER) && _MSC_VER >= 1800
+#define HAVE_EXPM1 1
+#endif
 
 /* Define to 1 if you have the `fabsl' function. */
 #define HAVE_FABSL 1
diff --git a/src/plfit/platform.h b/src/plfit/platform.h
index c98b8882..5ace053c 100644
--- a/src/plfit/platform.h
+++ b/src/plfit/platform.h
@@ -34,7 +34,7 @@
 
 __BEGIN_DECLS
 
-#ifdef _MSC_VER
+#if defined(_MSC_VER) && _MSC_VER < 1800
 #include <float.h>
 #include <math.h>
 
